src/kubernetes_cluster/proof/kubernetes_api_liveness.rs

```diff
@@ -223,6 +246,176 @@ pub open spec fn no_req_before_rest_id_is_in_flight(rest_id: RestId) -> StatePre
     }
 }
 
+/// Newly created message means it doesn't exist in in_flight in last state but exists in the current state.
+pub open spec fn every_new_in_flight_req_msg_is_expected(expected_messages: FnSpec(Message) -> bool) -> ActionPred<Self> {
```

Comment:
Maybe you can rename `_is_expected` to `_satisfies` and `expected_message` to `msg_pred`

Reply:
Done